app/controllers/students_controller.rb

```diff
@@ -24,27 +24,70 @@ def authorize_for_districtwide_access_admin
 
   def show
     student = Student.find(params[:id])
-    chart_data = StudentProfileChart.new(student).chart_data
+
+    serialized_student = log_timing('students#serialize_student_for_profile') do
```

Comment:
How about `students#show serialize_student_for_profile`?  That way it's clear it's a step within the `students#show` controller/action.

Reply:
👍